protocol/src/main/java/org/fido/iot/protocol/CryptoService.java

```diff
@@ -1546,6 +1494,10 @@ private Cipher aesTypeToCipher(int aesType)
       case Const.ETM_AES256_CTR:
         return Cipher.getInstance("AES/CTR/NoPadding");
 
+      case Const.ETM_AES128_CBC:
```

Comment:
Please replace usages of ETM_AES* constants with COSEAES* constants across the board.  Currently, COSEAES* constants are not in use.

Reply:
I don't think this is the right solution, as the COSEAES constants are EAT claims (and big negative numbers) but AES type values are COSEEncType, which is explicitly a reference to types in RFC8152, and those are small positive integers.

Reply:
I forgot that AES-GCM and CCM have been implemented, so not all references should be replaced as a result.

However, What I meant there is that the COSEAES* constants (large negative numbers) should be used for AESPlainType in ETMInnerBlock of 'Composed' Encrypted Message.
While the COSEEncType (small numbers) should be used in 'Simple' Encrypted Message.

Reply:
OK, fixed in d3053d5e.  I've also changed the name of the ETM_*_CCM/GCM constants to match the COSE* constants.  They never really had anything to do with Encrypt-Then-Mac anyway.